controllers/profiles/reconciler_dev_test.go

```diff
@@ -134,7 +134,7 @@ func Test_newDevProfile(t *testing.T) {
 
 	// check if the reconciliation ensures the object correctly
 	service = test.MustGetService(t, client, workflow)
-	assert.Equal(t, int32(defaultHTTPWorkflowPort), service.Spec.Ports[0].TargetPort.IntVal)
+	assert.Equal(t, int32(defaultHTTPWorkflowPortInt), service.Spec.Ports[0].TargetPort.IntVal)
```

Comment:
@ricardozanini  Nope, if removed the test fails because TargetPort is a IntOrString and the IntVal return a int32

Reply:
Therefore you can define this var as int32 directly since it's meant to be used with IntOrString.

Reply:
@ricardozanini 
We can't because we can't use int32 in the IntOrStr that is used in several part of the code 

- 2 HttpGet.Port IntOrStr 
- 1 Ports.TargetPort IntOrStr 
- 1 Containers.port int32 (here is converted without problem from the int value)

 the only place where is asked an explicit int32 without automatic conversion is this test, to remove here I have to add another const with same 8080 value but of type int32.
wdyt ?